### ISSUE TITLE
*MEDICINE* - Monoamine Oxidase Inhibitors

### ISSUE THREAD

**Issue by a-d-brown** [opensafely/codelist-development#69]
Hi,

Link to codelist: https://codelists.opensafely.org/builder/011286de/

This one seems quite straight forward.

No exclusions have been made from the BNF paragraph - Monoamine-oxidase inhibitors (040302).

All MAOI's are licenced for treatment of depression. Some are additionally licenced for anxiety and phobias. However their inclusion should not threaten the validity of the codelist given that there can be significant clinical overlap between depression and these conditions anyway.

Andrew

**Comment by orlamac** [comment]
@a-d-brown 
I agree with this list as described.  You might want to mention in metadata section that Moclobemide is included even though this is a reversible inhibitor of mono-amine oxidase.  The BNF includes them all in one category, but other psychiatric textbooks don't so it would be useful just to point out in the descriptive text.

**Comment by orlamac** [comment]
Link to dm&d codelist 
https://www.opencodelists.org/codelist/opensafely/monoamine-oxidase-inhibitors-dmd/2b785e16/

@brianmackenna and @richiecroker These codelists are now ready for sign off.

**Comment by HenryDrysdale** [comment]
@orlamac I've signed off the dm+d. I don't have access to the metadata for the pseudoBNF one but is it just the dm+d one that needs a second sign-off?
